Client/Ecosia/UI/EmptyBookmarksView.swift

```diff
@@ -0,0 +1,208 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+final class EmptyBookmarksView: UIView, NotificationThemeable {
+    
+    private enum UX {
+        static let TitleLabelFont = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 17, weight: .semibold))
+        static let SectionLabelFont = UIFontMetrics(forTextStyle: .callout).scaledFont(for: .systemFont(ofSize: 16))
+        static let LearnMoreButtonLabelFont = UIFontMetrics(forTextStyle: .callout).scaledFont(for: .systemFont(ofSize: 16))
+        static let ImportButtonLabelFont = UIFontMetrics(forTextStyle: .callout).scaledFont(for: .systemFont(ofSize: 16))
+        static let ImportButtonPaddingInset = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        static let LayoutMargingsInset: CGFloat = 12
+        static let ImportButtonBorderWidth: CGFloat = 1
+        static let ImportButtonCornerRadius: CGFloat = 20
+        static let TitleSpacerHeight: CGFloat = 24
+        static let SurroundingSpacerWidth: CGFloat = 32
+        static let InBetweenSpacerWidth: CGFloat = 6
+        static let SectionSpacerWidth: CGFloat = 36
+        static let SectionIconLabelSpacerWidth: CGFloat = 24
+        static let SectionEndSpacerHeight: CGFloat = 16
+    }
+
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = .localized(.noBookmarksYet)
+        label.textAlignment = .center
```

Comment:
Following up on other comment I'd expect to find `label.adjustsFontForContentSizeCategory = true` here

Reply:
also on the following labels/buttons